### ISSUE TITLE
Autoback not working again

### ISSUE THREAD

**Issue by Venger2005** [hakt0-r/homebridge-platform-linktap#6]
Hi. With your previous change, autoback was working for me. Recently the app has a major update and since then autoback is no longer working with this plug-in. Not sure if there were changes made to the API alongside the app revamp which have caused this behaviour?

Any help is appreciated as having to manually check now to make sure watering is going to occur as per my 7-day schedule, which kind of defeats the purpose of my linktap homebridge automation.

**Comment by hakt0-r** [comment]
Sure mate I'll have a look at this. I haven't played with my automation in a while I am using their app and have setup the timers in that. Have you tried their app, they have done some updates too in it.

**Comment by Venger2005** [comment]
I have been and am still using the app for my timers. But I also have an automation set up in Homebridge where if there is motion detected in a certain zone then the sprinkler will turn on for one minute and then turn off. Previously, after the changes you made, then after the automation turned off the watering mode would switch back from instant mode to the app-set 7-day schedule.  Since the app update though, the autoback function no longer works for me. 

Hope that makes sense!

**Comment by Venger2005** [comment]
Hi. Just checking if you managed to take a look at autoback? As I'm still having the same problem with it.  Any help is much appreciated.  Thank you.

**Comment by Venger2005** [comment]
Hi.  I think I figured this out.  I had uninstalled and reinstalled your plugin recently, so this got me thinking.  So I checked the index.js file to see if the changes you previously made were still there, and they weren't!  I had reinstalled the plugin through an npm install which installed version 0.0.9.  So not quite sure what's happened here but hopefully you would have more insight into this.
